_includes/includes/ui/keyboard-details.php

```diff
@@ -418,12 +431,18 @@ protected static function WriteKeymanWebBox() {
         return;
       }
 
+      // only show if the jsFilename property is present in the .keyboard_info
+      if(empty(self::$keyboard->jsFilename)) {
+        return;
+      }
+
       // only inject on desktop platforms
       $webDevice = self::GetWebDeviceFromPageDevice();
       if(!$webDevice) {
         return;
       }
 
+      $kmwid = self::GetWebKeyboardId();
```

Comment:
```suggestion
      $keymanWebId = self::GetWebKeyboardId();
```

Don't we want to get away from using `kmw` in our codebase? :smile: 
For l. 474 too